src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1324,21 +1670,43 @@ private static T SerializeAndDeserialize<T>(T value, string baseline, DataContra
             dcjs = (settings != null) ? new DataContractJsonSerializer(typeof(T), settings) : new DataContractJsonSerializer(typeof(T));
         }
 
+        Console.WriteLine("Testing input value : {0}", value);
```

Comment:
This should be for debugging purpose and has existed for a while. Now it should be changed to Debug.WriteLine. Apparently I've got a merge issue. I'll fix that.